x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/constants.ts

```diff
@@ -7,6 +7,9 @@
 import { PolicyFromES } from '../../../common/types';
 
 export const POLICY_NAME = 'my_policy';
+// navigation doesn't work for % with other special chars or sequence %25
+// https://github.com/elastic/kibana/pull/81664
+export const SPECIAL_CHARS_NAME = 'test?#';
```

Comment:
This file is only consumed in `app.test.ts`, and I find it's a bit easier to understand the tests if this value is colocated with the code that consumes it. Can we define it there until it needs to be used in multiple files, at which point we can extract it to a common location like this file?

Reply:
Also, what do you think of using the full set of reserved characters that the API allows?

```js
export const SPECIAL_CHARS_NAME = 'test?#$+=&@:';
```